app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -656,6 +656,12 @@ def build_db_options_for_select
     @protocols_arr = []
     DatabaseBackup.supported_depots.each { |depot| @protocols_arr.push(depot[1]) }
     @database_backup_options_for_select = @protocols_arr.sort
+    @regions_options_for_select = retrieve_regions
+  end
+
+  def retrieve_regions
```

Comment:
Just a minor thing: perhaps `retrieve_aws_regions` would be more appropriate routine name (to avoid confusion with miq regions).

Reply:
Agreed.